packages/node-sdk/src/client.ts

```diff
@@ -793,11 +795,14 @@ export class BucketClient {
     enableTracking = true,
     ...context
   }: ContextWithTracking): TypedFeatures {
-    const features = this._getFeatures({ ...context, enableTracking });
+    const options = { enableTracking, ...context };
+    checkContextWithTracking(options);
```

Comment:
Can probably be moved to the private getFeatures and shared across all callers. Also safer.